public/app/plugins/datasource/dashboard/datasource.ts

```diff
@@ -69,7 +73,7 @@ export class DashboardDatasource extends DataSourceApi<DashboardQuery> {
         sourceDataProvider?.setContainerWidth(500);
       }
 
-      const cleanUp = sourceDataProvider!.activate();
+      const cleanUp = activateInActiveParents(sourceDataProvider!);
```

Comment:
think it's time to rename this util to 

* activateSceneObjectAndParentTree 

The way I named it makes you think it only activates the parents :)